doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -1,3 +1,262 @@
+
+## `econia_sdk.types`
+
+This package contains various enum types useful for parsing and referring to important values. Note that Move doesn't have Enum types so unlike most of the above, these do not map directly to Move. However, each value in each enum is associated with a constant that exists in the Move code.
+
+# Other Contents
+
+## `examples.trade`
+
+This is a script that makes use of both view functions and entry functions to perform a few scenarios in the exchange for the user. After set-up, it should be automatic (except for hitting enter to proceed the script). On display are things like registering a market, creating a market account and funding it, placing limit/market orders under various conditions as well as cancelling them. It's useful as a way to gain an understanding of what different fields/parameters mean and in general how to use the Python SDK. **Running/reading this example script and understanding what it does is recommended before anyone trades real money with the SDK.**
+
+### Running the Example Script ("Trading From Scratch")
+
+You're recommended to run a local Aptos node and faucet first; this prevents any rate-limitting issues from preventing your progress. Install the Aptos CLI & run your local node/faucet:
+
+```bash
+brew install aptos # only if necessary
+mkdir aptos && cd aptos
+aptos node run-local-testnet --with-faucet
+```
+
+In another terminal, run the following:
+
+```bash
+export APTOS_NODE_URL=http://0.0.0.0:8080/v1
+export APTOS_FAUCET_URL=http://0.0.0.0:8081
+```
+
+You'll also have the option of entering these as prompts of the script, but the environment variable is preferred because it's easier to run multiple times. It's time to deploy our own Econia Faucet to the local chain:
+
+```bash
+git clone https://github.com/econia-labs/econia.git # only if necessary
+cd ./econia/src/move/faucet
+aptos init --profile econia_faucet_deploy # enter "local" for the chain
+export FAUCET_ADDR=<ADDR-FROM-ABOVE>
+# deploy the faucet (all one command)
+aptos move publish \
+        --named-addresses econia_faucet=$FAUCET_ADDR \
+        --profile econia_faucet_deploy \
+        --assume-yes
+```
+
+We also need to deploy an Econia exchange:
+```bash
+cd ./econia/src/move/econia
+aptos init --profile econia_exchange_deploy # enter "local" for the chain
+export ECONIA_ADDR=<ADDR-FROM-ABOVE>
+# deploy the faucet (all one command)
+aptos move publish \
+        --override-size-check \
+        --included-artifacts none \
+        --named-addresses econia=$ECONIA_ADDR \
+        --profile econia_exchange_deploy \
+        --assume-yes
+```
+
+It's to run the script! Setting our environment variables will have cleared the initial setup prompts for us. In order to run, install Poetry then install dependencies and run the script:
+
+```bash
+curl -sSL https://install.python-poetry.org | python3 - # only if necessary
```

Comment:
Will you please change this to a brew install to be consistent with other installs?